bidwire/bidwire_settings.py

```diff
@@ -55,6 +57,11 @@ def get_recipients_list(env_var_name, default_recipients='bidwire-logs@googlegro
         'notifier': CityOfBostonNotifier(),
         'recipients': EMAIL_RECIPIENTS
     },
+    Document.Site.BOSTON_NOTICES: {
+        'scraper': BostonNoticeScraper(),
+        'notifier': BostonNoticeNotifier(),
+        'recipients': EMAIL_RECIPIENTS
```

Comment:
This is going to need to be set to a different recipient variable. You can add a new one specifically for `BOSTON_NOTICES`, same as e.g. the `KNOX_COUNTY_COMMISSION_AGENDA_RECIPIENTS` recipient list above.